dvc/repo/experiments/executor/base.py

```diff
@@ -275,6 +277,13 @@ def save(
         else:
             os.chdir(dvc.root_dir)
 
+        include_untracked = include_untracked or []
+        include_untracked.extend(_collect_top_level_metrics(dvc))
+        include_untracked.extend(_collect_top_level_params(dvc))
```

Comment:
@skshetry what is the purpose of `index._plot_sources`? Is it used by Studio?

Should we have `index._metric_sources` and `index._param_sources` since I now need to use the `_collect` methods in more places?